orchestra/cmds/clone.py

```diff
@@ -12,7 +12,6 @@ def install_subcommand(sub_argparser):
                                           parents=[execution_options],
                                           )
     cmd_parser.add_argument("component", help="Name of the component to clone")
-    cmd_parser.add_argument("--no-force", action="store_true", help="Don't force execution of the root action")
```

Comment:
Why are we dropping this? Now if I try to clone an already cloned component, it will fail.

Reply:
The new scheduling algorithm deprecated `--no-force`, it already did nothing.
The actions requested by the user are always performed, of course if you already cloned a component cloning it again will fail. 
I noticed that there's also another `--no-force` for the configure command which should also be removed.
If `--no-force` is needed I can reintroduce it with a slight modification to Executor